src/event_listener.rs

```diff
@@ -155,17 +153,16 @@ impl IngestionInfo {
     }
 }
 
-#[repr(transparent)]
-pub struct WriteStallInfo(DBWriteStallInfo);
+pub struct WriteStallInfo(crocksdb_writestallinfo_t);
 
 impl WriteStallInfo {
     pub fn cf_name(&self) -> &str {
         unsafe { fetch_str!(crocksdb_writestallinfo_cf_name(&self.0)) }
     }
-    pub fn cur(&self) -> WriteStallCondition {
+    pub fn cur(&self) -> crocksdb_writestallcondition_t {
```

Comment:
All public functions/traits of `src/xxx.rs` should be unchanged, otherwise `TiKV` or other users may change code dramatically. Consider wrapping `crocksdb_writestallcondition_t` with `WriteStallCondition`, just like how `WriteStallInfo` does. Same for other types.

Reply:
OK